tests/integrationtest/t/executor/simple.test

```diff
@@ -165,6 +165,7 @@ disconnect conn1;
 # TestFlushPrivileges
 drop user if exists 'testflush'@'localhost';
 CREATE USER 'testflush'@'localhost' IDENTIFIED BY '';
+SHOW GRANTS FOR 'testflush'@'localhost';
 UPDATE mysql.User SET Select_priv='Y' WHERE User="testflush" and Host="localhost";
 connect (conn1, localhost, testflush,,);
 --error 1142
```

Comment:
This is a behavior change.
In the past, all the user data are in-memory, and login will not trigger reload.
Now login will trigger the reload if the user is not loaded,  login make the user get the latest privileges.

To test the old behavior, `SHOW GRANTS FOR 'testflush'@'localhost';` this line is added.
This cause the user data been loaded
Then login find the user and does not trigger reload.